client-telemetry-reporter-plugin/kafka/README.md

```diff
@@ -0,0 +1,209 @@
+<!-- title: How to use KIP-714 for centralized monitoring of Kafka Clients -->
+<!-- description: In this tutorial, learn how to use KIP-714 for centralized monitoring of Kafka Clients. -->
+
+# How to use KIP-714 for centralized monitoring of Kafka Clients
+
+Kafka's KIP-714 introduces a new capability that allows applications to send client metrics to the
+broker for centralized monitoring. The broker can subsequently relay these metrics to a remote monitoring
```

Comment:
little bit of a nit on who does what.... "allows applications to send client metrics to the broker" IMO implies that the app / client has a responsibility. Isn't it more accurate to say something like "KIP-714 introduces a new capability that allows the Kafka broker to centrally track client metrics on behalf of applications. The broker can subsequently relay ..."

Reply:
OK, I have corrected the text.